src/mscorlib/src/System/Threading/Tasks/ConcurrentExclusiveSchedulerPair.cs

```diff
@@ -251,16 +251,6 @@ private void FaultWithTask(Task faultedTask)
         /// </summary>
         public TaskScheduler ExclusiveScheduler { get { return m_exclusiveTaskScheduler; } }
 
-        /// <summary>Gets the number of tasks waiting to run concurrently.</summary>
-        /// <remarks>This does not take the necessary lock, as it's only called from under the debugger.</remarks>
-        [SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
-        private int ConcurrentTaskCountForDebugger { get { return m_concurrentTaskScheduler.m_tasks.Count; } }
-
-        /// <summary>Gets the number of tasks waiting to run exclusively.</summary>
-        /// <remarks>This does not take the necessary lock, as it's only called from under the debugger.</remarks>
-        [SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
-        private int ExclusiveTaskCountForDebugger { get { return m_exclusiveTaskScheduler.m_tasks.Count; } }
```

Comment:
Removing all of these "ForDebugger" properties breaks the debugger display attributes.